roktux/src/main/java/com/rokt/roktux/RoktUxConfig.kt

```diff
@@ -83,6 +85,13 @@ class RoktUxConfig private constructor(
          */
         fun viewStateConfig(viewStateConfig: ViewStateConfig) = apply { this.viewStateConfig = viewStateConfig }
 
+        /**
+         * Sets the edgeToEdgeDisplay.
+         *
+         * @param edgeToEdgeDisplay The edgeToEdgeDisplay configuration to use in the UX.
+         */
+        fun edgeToEdgeDisplay(edgeToEdgeDisplay: Boolean) = apply { this.edgeToEdgeDisplay = edgeToEdgeDisplay }
```

Comment:
So we have to update the UXHelper documentation on the release?
Do you want to include it in the changelog.md now, not to forget?

Reply:
yes updated and NI-581 ticket is created for documentation update.